ui/components/ui/sender-to-recipient/index.scss

```diff
@@ -107,7 +107,7 @@
         justify-content: center;
         flex: 1;
         border-radius: 4px;
-        box-shadow: 0 2px 4px rgba(0, 0, 0, 0.08);
+        box-shadow: var(--shadow-size-xs) var(--color-shadow-default);
         padding: 6px;
         background: var(--color-background-default);
         cursor: pointer;
```

Comment:
## Before
<img width="893" alt="Screen Shot 2022-07-17 at 1 39 29 PM" src="https://user-images.githubusercontent.com/8112138/179424013-7c748086-759c-44da-b74a-8e1fe7209792.png">
<img width="911" alt="Screen Shot 2022-07-17 at 1 39 59 PM" src="https://user-images.githubusercontent.com/8112138/179424016-fd12cd6e-abd2-4334-9d34-cc2ee6d82b36.png">

## After
<img width="903" alt="Screen Shot 2022-07-17 at 1 38 18 PM" src="https://user-images.githubusercontent.com/8112138/179424024-84bba9bc-7b38-4fe6-b98d-8565e305535c.png">
<img width="907" alt="Screen Shot 2022-07-17 at 1 38 01 PM" src="https://user-images.githubusercontent.com/8112138/179424023-ba23ee75-cb70-4337-ad87-2349a17d0d91.png">

Reply:
Is the shadow only applied in dark mode here? for light mode it looks like the before and after have the same values